batchupload/uploader.py

```diff
@@ -2,13 +2,14 @@
 # -*- coding: utf-8  -*-
 #
 from __future__ import unicode_literals
-from builtins import open
 import batchupload.common as common
 import batchupload.prepUpload as prepUpload
+from batchupload.make_info import make_info_page
 import os
 import pywikibot
 
 FILE_EXTS = ('.tif', '.jpg', '.tiff', '.jpeg')
```

Comment:
I'm guessing these are the formats that we're going to encounter in the KMB set, and that's why we're not making it more versatile with e.g. .png? I'm wondering because while the background of this patch is KMB, we're ultimately modifying the base tool, that could be used for other stuff in the future.